sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/base/BaseFragment.kt

```diff
@@ -0,0 +1,64 @@
+package co.nimblehq.sample.compose.ui.base
+
+import android.os.Bundle
+import android.view.*
+import androidx.annotation.CallSuper
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.*
+import co.nimblehq.sample.compose.ui.common.Toaster
+import co.nimblehq.sample.compose.ui.userReadableMessage
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+abstract class BaseFragment : Fragment(), BaseFragmentCallbacks {
```

Comment:
Should we keep the name as `BaseComposeFragment.kt` & `BaseComposeFragmentCallbacks.kt` for now? 🙇 

I think it will be more aligned with `template` this way 🙏

Reply:
@Tuubz Sure thing, but how about `HomeFragment` and `HomeViewModel`? 🤔

Reply:
Oh, good point. 

You mean keep it as `HomeComposeFragment.kt` & `HomeComposeViewModel.kt`, right? 

Yeah, let's do that 🙇

Reply:
@Tuubz Updated 🚀